packages/amplify-cli/src/context-extensions.ts

```diff
@@ -225,7 +225,7 @@ function blue(message: string): void {
   console.log(colors.blue(message));
 }
 
-function fancy(message: string): void {
+function fancy(message?: string): void {
```

Comment:
Is there a reason only this function has message as optional?

Reply:
There was some place where a potentially undefined value was being passed in (unrelated to this PR) and this was the easiest way to get TS to stop complaining